content/2017/05/27/liveqa-biz.rst

```diff
@@ -0,0 +1,94 @@
+public: yes
+author: sondra
+type: event
+tags: [Event, Q&A, Tips, Slack, Startup, Business Development]
+image:
+  - src: 'blog/2017/liveqa-biz/desk-mobile.jpg'
+summary: |
+  Are you a startup founder
+  looking to develop a new web app
+  or a company interested in expanding your web services?
+  Here's your chance
+  to ask all those questions
+  that keep you up at night.
+
+  .. callmacro:: content.macros.j2#link_button
+    :url: 'http://friends.oddbird.net'
```

Comment:
Is it possible to link directly to a specific channel? Or how are we actually telling users which Slack channel to join?